### PR TITLE
Functional tests - Add new test 'Enable/disable demo mode'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      |  Add new test 'Enable/disable demo mode'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | https://github.com/nesrineabdmouleh/PrestaShop/actions/runs/4045722007
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   | no

### CODE REVIEW

**Review by SD1982 — DISMISSED** [review]

File: tests/UI/pages/BO/dashboard/index.ts [hunk 1]
```diff
@@ -16,6 +23,38 @@ class Dashboard extends BOBasePage {
...
+    }
+  }
+
+  /**
+   * Get sales score
+   * @param page {Page} Browser tab
+   */
+  async getSalesScore(page: Page): Promise<number> {
+    const text = await this.getTextContent(page, this.salesScore);
+    const salesScore = text.match(/\d+(\.\d+)?/g).toString().replace(',', '');
```
Comment (Progi1984) [inline tests/UI/pages/BO/dashboard/index.ts]:
```suggestion
    const salesScore: string = text.match(/\d+(\.\d+)?/g).toString().replace(',', '');
```

File: tests/UI/pages/BO/dashboard/index.ts [hunk 2]
```diff
@@ -16,6 +23,38 @@ class Dashboard extends BOBasePage {
...
+
+    return (parseFloat(salesScore));
```
Comment (Progi1984) [inline tests/UI/pages/BO/dashboard/index.ts]:
```suggestion
    return parseFloat(salesScore);
```

**Review by Progi1984 — COMMENTED** [review]
Two small comments and it's perfect.

**Review by Progi1984 — DISMISSED** [review]

File: tests/UI/package.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "// Functional BO": "Scripts to run functional BO tests",
     "test:functional:BO": "TEST_PATH='functional/BO/**/*.{ts,js}' npm run test:specific",
     "test:functional:BO:login": "TEST_PATH='functional/BO/00_login/**/*.{ts,js}' npm run test:specific",
+    "test:functional:BO:dashboard": "TEST_PATH='functional/BO/01_dashboard/**/*.{ts,js}' npm run test:specific",
```
Comment (boubkerbribri) [inline tests/UI/package.json:30]: should be added to nightly tests ?

File: tests/UI/pages/BO/dashboard/index.ts [hunk 4]
```diff
@@ -16,6 +23,38 @@ class Dashboard extends BOBasePage {
...
+    }
+  }
+
+  /**
+   * Get sales score
+   * @param page {Page} Browser tab
+   */
+  async getSalesScore(page: Page): Promise<number> {
+    const text = await this.getTextContent(page, this.salesScore);
+    const salesScore : string = text.match(/\d+(\.\d+)?/g).toString().replace(',', '');
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/dashboard/index.ts]:
```suggestion
    const salesScore: string = text.match(/\d+(\.\d+)?/g).toString().replace(',', '');
```

File: tests/UI/package.json (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/package.json:30]: yes :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**